cmd/virsnap/root.go

```diff
@@ -3,43 +3,60 @@
 // the "LICENSE" file in this repository.
 
 // Package cmd implements the handlers for the different command line arguments.
```

Comment:
Please change "cmd" in the comment to "main".

Reply:
Fixed